### ISSUE TITLE
Magenta not getting transpiled to ES5?

### ISSUE THREAD

**Issue by cifkao** [notwaldorf/example-magenta-in-ts#4]
From what you wrote [here](https://github.com/magenta/magenta-js/issues/432), the output should be ES5 compatible. However, the magenta code doesn't seem to get transpiled to ES5, as the output contains arrow functions which, if I'm not mistaken, are ES6:

![image](https://user-images.githubusercontent.com/8046580/77955297-38354280-72d0-11ea-9718-82d490157eec.png)

**Comment by notwaldorf** [comment]
I don't think this project is being told to transpile to ES5, and I definitely did not test anything about transpilation in it. It's supposed to show how to use Magenta in a webpack/module-y project. I didn't promise in my original message that this particular project is being transpiled to ES5, but rather that it can use Magenta as an import, which I thought is what you were asking ("How do I make 4) work")

In any case, at least the module field [here](https://github.com/notwaldorf/example-magenta-in-ts/blob/master/tsconfig.json#L21) looks wrong, and it should be 'commonjs', like it is in the actual [config](https://github.com/magenta/magenta-js/blob/master/music/tsconfig.es5.json) we use for transpiling magenta to es5. That being said, i just checked the magenta es5 [build](https://cdn.jsdelivr.net/npm/@magenta/music@%5E1.0.0) and it also has => so uhhhh I don't know what to tell you. I think IE11 might be the only thing at this point that doesn't have ES6 (and fat arrows etc), and we have no real way of testing any of the magenta stuff in IE11, so I can't actually guarantee anything works in IE11.

**Comment by cifkao** [comment]
Well I was already using Magenta as an import, I was just wondering how to use it correctly to have ES5 compatibility. So I guess the answer is that it's not clear. But yeah, maybe it's not a huge problem anymore.
